core/lib/stack-packs.js

```diff
@@ -63,10 +63,12 @@ const stackPacksToInclude = [
 
 /**
  * Returns all packs that match the stacks found in the page.
- * @param {LH.Artifacts['Stacks']} pageStacks
+ * @param {LH.Artifacts['Stacks']|undefined} pageStacks
  * @return {LH.RawIcu<Array<LH.Result.StackPack>>}
  */
 function getStackPacks(pageStacks) {
+  if (!pageStacks) return [];
```

Comment:
We can reach this condition on error reports now

Reply:
Isn't that just because of removing the initial values in initializeBaseArtifacts ? Maybe add them back?

Reply:
We shouldn't use `initializeBaseArtifacts` to subtly create a default value for some artifacts but not others IMO